app/src/main/res/layout/screen_patient_prescribed_drugs_entry.xml

```diff
@@ -39,4 +39,22 @@
       android:textAppearance="@style/Clinic.V2.TextAppearance.Button1" />
   </org.simple.clinic.widgets.PrimarySolidButtonWithFrame>
 
+  <org.simple.clinic.widgets.PrimarySolidButtonWithFrame
```

Comment:
I missed it during the initial reveal, looks like we have added 2 `PrimarySolidButtonWithFrame` in this screen which should not be the case. Since both these buttons have the same colour and share the same frame background, we can wrap these two buttons in a `ConstraintLayout` and set its background color and just show/hide buttons individually. You can take a look at the `screen_patient_summary` for a example.